src/tests/test_gcl_page.py

```diff
@@ -36,7 +36,9 @@ def test_donate(browser):
 	page = GCLPOM(browser)
 	page.load()
 	page.test_donate()
-	assert browser.current_url == 'https://www.paypal.me/girlscodelincoln'
+	url = 'https://www.paypal.me/girlscodelincoln'
+	url2 = 'https://www.paypal.com/paypalme2/girlscodelincoln'
```

Comment:
Why is this necessary?

Reply:
URL is the correct code and when running outside the container it normally takes you to. However for some reason, unknown to me the 2nd url is hit most times inside the container. So in order to get it to pass inside and outside the container I had to ad the OR.